tests/compliance/date/test_date_compliance.py

```diff
@@ -22,6 +22,7 @@
 
 import pandas
 from pandas.tests.extension import base
+import pandas._testing as tm
```

Comment:
There's actually a public module for this. Let's use that.

https://pandas.pydata.org/docs/reference/api/pandas.testing.assert_series_equal.html#pandas-testing-assert-series-equal